CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ set (CMAKE_MODULE_PATH
 set (BOND_ENABLE_GRPC
     "TRUE"
     CACHE BOOL "If FALSE, then do not build gRPC integration")
+set (BOND_SKIP_GRPC_LIB_COMPILE
+    "FALSE"
```

Comment:
@chwarr Should we set this to be default to `TRUE` and `BOND_ENABLE_GRPC` to `FALSE`?

Reply:
That would be a breaking change. I'm not sure the benefit is large enough. I have a different approach that sort of avoids this.